.travis.yml

```diff
@@ -7,10 +7,6 @@ git:
 node_js:
   - "10"
 
-cache:
-  directories:
-    - node_modules # NPM packages
```

Comment:
Have to remove this caching feature, otherwise the build will fail due to using old dependencies.

This appears to be a known issue for travis-ci https://github.com/travis-ci/travis-ci/issues/3758

Reply:
@zflamig is this an issue?

Reply:
@abgeorge7  I think Mingfei will recover this change when we merge ui-component and guppy

Reply:
restored